### DISCUSSION TITLE
Can't delete PKGs that are cancelled downloading

### DISCUSSION THREAD

**Discussion by AForceGaming** [bucanero/pkgi-ps3#58] [Q&A] [answered]
So, I've got a PS3 Super Slim with 35GB and that's a very limited storage. Let's say when downloading 5GB game or whatever, and I cancelled at 50% it doesn't really cancel the download it has removed 2.5GBs from the storage. **Why can't completely erase PKGs that are paused or cancelled?** . So you can't delete it until the download is complete.
Anyways I hope an update comes out soon to fix this issue, _please fix!_

**Comment by bucanero** [comment, marked as answer]
if you're using `direct download` mode, partial files are kept in the HDD in case you want to continue the download later.

to delete partial downloads, just erase the `.pkg` files in `/dev_hdd0/tmp/pkgi/`

btw, you can upgrade your PS3 with a bigger hard drive. All models support it.
